examples/plugins/inspec-resource-lister/test/functional/inspec_resource_lister_test.rb

```diff
@@ -0,0 +1,110 @@
+# Functional Tests for Example Plugin, Resource Lister
+
+# Functional tests are used to verify the behavior of the plugin are as expected, to a user.
+# Functional tests generally do not have inside knowledge of how the plugin works.
+
+# Include our test harness
+require_relative '../helper'
+
+# Because InSpec is a Spec-style test suite, we're going to use MiniTest::Spec
+# here, for familiar look and feel. However, this isn't InSpec (or RSpec) code.
+describe 'inspec list-resources core' do
+  # Our helper.rb locates this library from the InSpec install that
+  # Bundler installed for us. If we want its methods, we still must
+  # import it.  Including it here will make it available in all child
+  # 'describe' blocks.
+  include CorePluginFunctionalHelper
+
+  # When thinking up scenarios to test, start with the simplest.
+  # Then think of each major feature, and exercise them.
+  # Running combinations of features makes sense if it is very likely,
+  # or a difficult / dangerous case.  You can always add more tests
+  # here as users find subtle problems.  In fact, having a user submit
+  # a PR that creates a failing functional test is a great way to
+  # capture the reproduction case.
+  # The simplest case:
+  describe "when run without an argument" do
```

Comment:
RuboCop would complain about double quotes without interpolation. The parent `describe` block is correct...but the rest seem to use double quotes. I can ignore it, but I'd like to see the same used within this file.

Reply:
Please use the team's agreed-upon guidelines for where and how we apply style rules.